test/unit/ui-ingress.bats

```diff
@@ -60,61 +60,56 @@ load _helpers
 }
 
 @test "ui/Ingress: exposes single port 80 when global.tls.enabled=false" {
```

Comment:
Helm 3 does not support the `--kube-version` flag yet so we can't test this permutation until the linked PR is merged.